ui/js/models/perf/alerts.js

```diff
@@ -25,16 +25,16 @@ treeherder.factory('PhAlerts', [
 
             // for talos only, automatically add related branches
             if (performanceFrameworkId === 1) {
-                _.forEach(thPerformanceBranches, function(performanceBranch) {
-                    if (performanceBranch !== alertRepository) {
-                        url += "&series=[" + [performanceBranch, signature, 0] + "]";
-                    }
-                });
                 if (alertRepository === "mozilla-beta") {
-                    url += "&series=[" + ["mozilla-aurora", signature, 0] + "]";
+                    url += "&series=[" + ["mozilla-inbound", signature, 0] + "]";
+                } else {
+                    _.forEach(thPerformanceBranches, function(performanceBranch) {
+                        if (performanceBranch !== alertRepository) {
+                            url += "&series=[" + [performanceBranch, signature, 0] + "]";
+                        }
+                    });
```

Comment:
Let's confirm with @jmaher but I think we can actually just use the default branches here. If we're going to show mozilla-inbound why would we not also want to show autoland? I suspect this was just an oversight (in which case we can just leave this as-is).